packages/123done/server.js

```diff
@@ -1,3 +1,5 @@
+require('module-alias/register');
```

Comment:
Interesting! Why was this the only module that needed the alias registering?

Reply:
123done is a pure JS package that isn't compiled, so the path aliases used in `libs/` won't be resolved correctly for import without this. An alternative would be applying TS to a pure JS project, which is doable but this seemed less effort.